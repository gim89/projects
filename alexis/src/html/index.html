<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=1144">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Alexis</title>
  <link rel="stylesheet" href="css/styles.css">
</head>
<body>
  //= component/scroll.html
  //= component/header.html
  //= component/about.html
  //= component/focus.html
  //= component/part-one.html
  //= component/portfolio.html
  //= component/team.html
  //= component/part-two.html
  //= component/steps.html
  //= component/pricing.html
  //= component/testimonials.html
  //= component/select.html
  //= component/blog.html
  //= component/newsletter.html
  //= component/touch.html
  //= component/footer.html
  <script src="js/scripts.js"></script>
</body>
</html>