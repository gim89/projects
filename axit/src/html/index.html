<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=1200">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>AX IT</title>
  <link rel="stylesheet" href="css/styles.css">
</head>
<body>
  //= component/header.html
  //= component/social.html
  //= component/features.html
  //= component/process.html
  //= component/pricing.html
  //= component/testimonial.html
  //= component/custom.html
  //= component/contact.html
  //= component/footer.html
</body>
</html>