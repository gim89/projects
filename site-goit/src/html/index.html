<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <meta http-equiv="X-UA-Compatible" content="ie=edge">

        <link rel="stylesheet" href="css/bootstrap.css">
    	<link rel="stylesheet" href="css/styles.css">

        <title>Home</title>
    </head>
    <body>
        <div class="container">
            //= base/header.html
            <main class="main">
                //= home/home.html
            </main>
            //= base/footer.html
        </div>
        <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js"></script>
    </body>
</html>
