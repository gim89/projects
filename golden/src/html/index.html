<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=940">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>golden</title>
  <link rel="stylesheet" href="css/styles.css">
</head>
<body>
  //= component/header.html
  //= component/service.html
  //= component/portfolio.html
  //= component/about.html
  //= component/team.html
  //= component/logos.html
  //= component/contact.html
  //= component/footer.html
</body>
</html>